contracts/p1/mixins/RecollateralizationLib.sol

```diff
@@ -391,9 +414,36 @@ library RecollateralizationLibP1 {
             }
         }
 
+        // Special-case RToken
+        {
+            RTokenAsset rTokenAsset = RTokenAsset(
+                address(ctx.ar.toAsset(IERC20(address(ctx.rToken))))
+            );
+            uint192 bal = rTokenAsset.bal(address(ctx.bm));
+
+            if (
+                ntpCtx.rTokenPrice.high > 0 &&
```

Comment:
why use rTokenPrice.high here but rTokenLotPrice.low below?

Reply:
This is as it was before, and I believe it's correct. It's for the case where the RToken price is unknown ([0, FIX_MAX]). In that case it's correct to use 0 as the sellPrice and allow the auction to handle price discovery, as long as the `isEnoughToSell()` call says the low lotPrice is sufficiently large.

Now, you could argue the `high > 0` check here is redundant with the `isEnoughToSell` call...The only case I can see where the `high > 0` check would differentiate itself would be if there were a mistake in the lotPrice, such that price() were [0, 0] but lotPrice() were nonzero.